test/boostTest.cpp

```diff
@@ -110,21 +110,6 @@ int main( int argc, char* argv[] )
 	try
 	{
 		framework::init(fake_init_func, argc, argv);
-#if BOOST_VERSION >= 106000
```

Comment:
it is not my code neither. It was here before and it is just appears that it might not even be needed.